site/docs/components/infinite-scroll.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Infinite Scroll with a “View more” button
+navTitle: Infinite Scroll (not built)
+summaryParagraph: Infinite Scroll separates large bodies of content into separate sections. You can access each extra section via a "View more" button.
+tags: ["Infinite scrolling", "Continuous scroll", "Load more", "Async loading"]
+needToKnow:
+  - Perform usability testing to decide where to split content into pages.
```

Comment:
Does this mean how much content to display before implementing an infinite scroll? I know one thing we were kinda just guessing it how many comments to display in each "Show more" page. We went with 50 FWIW.